bugbot/rules/assignee_no_login.py

```diff
@@ -100,13 +103,19 @@ def add_action(self, bug):
         default_assignee = self.default_assignees[prod][comp]
         autofix = {"assigned_to": default_assignee}
 
+        is_old_priority = bug["is_old_priority"]
+
```

Comment:
Why do we need to assign it to a variable here?

Reply:
We don't really need it, removed it here: 737ccb2579dd034fc3c7a0ad12004fbdce6fe8aa